scripts/cnn_variant_wdl/run_happy.wdl

```diff
@@ -0,0 +1,148 @@
+# Run the hap.py VCF evaluation over input vcfs given a validated truth vcf and confidence region
+workflow HappyWorkflow {
+    File vcf_fofn           # Text file with list of VCFs to evaluate with hap.py
+
+    File reference_fasta
+    File reference_dict
+    File reference_fasta_index
+
+    File truth_vcf
+    File truth_vcf_index
+    File truth_bed
+
+    File rscript
+
+    Int? preemptible_attempts
+    Int? disk_space
+    Int? mem_gb
+    Int? cpu
+
+    call RunHappy {
+        input:
+            vcf_fofn = vcf_fofn,
+            truth_vcf = truth_vcf,
+            truth_vcf_index = truth_vcf_index,
+            truth_bed = truth_bed,
+            reference_fasta = reference_fasta,
+            reference_dict = reference_dict,
+            reference_fasta_index = reference_fasta_index,
+            cpu = cpu,
+            mem_gb = mem_gb,
+            disk_space = disk_space,
+            preemptible_attempts = preemptible_attempts
+    }
+
+    call RunHappyPlots{
+        input:
+            happy_outputs = RunHappy.happy_outputs,
+            rscript = rscript,
+            cpu = cpu,
+            mem_gb = mem_gb,
+            disk_space = disk_space,
+            preemptible_attempts = preemptible_attempts
+    }
+
+    output {
+        RunHappy.*
+        RunHappyPlots.*
+    }
+}
+
+task RunHappy {
+    File vcf_fofn
+    Array[File] vcf_files=read_lines(vcf_fofn)
+
+    File reference_fasta
+    File reference_dict
+    File reference_fasta_index
+
+    File truth_vcf
+    File truth_vcf_index
+    File truth_bed
+
+    # Runtime parameters
+    Int? mem_gb
+    Int? preemptible_attempts
+    Int? disk_space
+    Int? cpu
+    Boolean use_ssd = true
```

Comment:
I'm not sure this will make a performance difference.

Reply:
probably not, going to HDD